contribute/README.md

```diff
@@ -7,6 +7,7 @@ This directory contains guides for contributors to the Grafana project.
 - [Developer guide](developer-guide.md)
 - [Triage issues](triage-issues.md)
 - [Merge a pull request](merge-pull-request.md)
+- [Backend documentation](/contribute/backend/)
```

Comment:
I'd suggest including this as a link inside `developer-guide.md` instead so that we can keep the developer resources grouped instead of having multiple ones at this level.